### ISSUE TITLE
tor_lookup pipeline function returns always false

### ISSUE THREAD

**Issue by mudrunkar** [Graylog2/graylog-plugin-threatintel#115]
## Expected Behavior

I tested the pipeline function `tor_lookup` with several tor exit node's addressess, but all I got was `{"threat_indicated":false}` response. Does it work for anyone?

## Current Behavior

Only negative response `{"threat_indicated":false}` is returned as a result of tor lookup, there is no error message in the log.

## Steps to Reproduce (for bugs)

I tested the lookup function with the following IP addressess where I would expect positive tor lookup result:

![obrazek](https://user-images.githubusercontent.com/35534668/35516022-f312e764-050a-11e8-9ae4-79658d648021.png)

but I got false in case of all of these IP addressess. 

Pipeline function used:

```
rule "add_tor_lookup"
when 
    has_field("source_address")
then
    let tor_lookup = tor_lookup(to_string($message.source_address));
    set_field("tor_lookup", tor_lookup);
end 
```

Does the (now built-in) tor lookup plugin work for someone? Maybe I'm doing something wrong?

## Your Environment

* Graylog Version: 2.4.1
* Elasticsearch Version:
* MongoDB Version:
* Operating System:
* Browser version:

**Comment by dio99** [comment]
will it not be fixed in 2.5?
// Anders

**Comment by danotorrey** [comment]
Hi @mudrunkar @stamfest @ion-storm @dio99,
This will be fixed in Graylog version 3.0 coming out next month, but there is a solid workaround:

1) Verify that a Tor Exit Node Lookup Table is set up.
2) Add the following pipeline rule that uses the lookup table by it's name:

```
rule "Is from a Tor Exit Node: src_addr"
when 
    has_field("src_addr")
then
    let tor_lookup = lookup( "name-of-tor-exit-node-lookup-table", to_string($message.src_addr));
    set_field("from_tor_exit_node", is_not_null(join(tor_lookup.node_ids)));
end
```

See this [docs](http://docs.graylog.org/en/2.5/pages/lookuptables.html#setup) page for general instructions for setting up a lookup table. Please note that you may need to also enable Tor Lookups in System > Configurations > Threat Intelligence Lookup Configuration > Tor Exit Nodes.

**Comment by dio99** [comment]
complains about function join in graylog version 2.5
//Anders
